tera-templates/core/about.html

```diff
@@ -28,6 +28,12 @@ <h1 id="crate-title">About Docs.rs</h1>
             {%- endif -%}
         </p>
 
+        <p>
+            Docs.rs builds crates with environment variable <code>DOCS_RS</code> set to
+            `1`, which enables the crate to detect docs.rs and build the
```

Comment:
I'll just add it myself if you don't mind :P
```suggestion
            <code>1</code>, which enables the crate to detect docs.rs and build the
```